dbt-cta/monitoring/models/0_staging/workflows_execution_logs/stg_workflows_execution_logs.sql

```diff
@@ -0,0 +1,30 @@
+with
+    source as (select * from {{ source("workflows_logs", "executions") }}),
+    unpack_nested_fields as (
+        select
+            * except (logName, resource, textPayload, jsonpayload_type_executionssystemlog),
+            JSON_EXTRACT_SCALAR(jsonpayload_type_executionssystemlog.start.argument,'$.partner_name') as partner_name,
```

Comment:
so cool! gotta go through and add this arg into all our currently running workflows!

Reply:
> So the Overview tab will only show workflows in the sheet but the Workflow Executions tab shows all the executions...?

Currently the Overview shows the status of the most recent run per workflow_id * partner per day - it's not clear that's the best representation of "Workflow Sync Status", given that workflows might run at arbitrary times (unlike DAGs that run once daily). That's part of why I figured the "Workflow Executions" view might be useful, because it shows a row per each log from a workflow execution (which so far seems to involve two log messages, one when the sync starts and has status `ACTIVE` and one when it completes with a status of `FAILED` or `SUCCEEDED`).

I'll plan to walk the team through the dbt in Eng Weekly. It's not the easiest setup to dive into without some guidance!